src/config/Components.jsx

```diff
@@ -1,5 +1,6 @@
-import { PreviewImage } from '@plone/volto/components';
+import { App, PreviewImage } from '@plone/volto/components';
```

Comment:
I'd start using full paths as default, instead of using the indexes... for code splitting, right?